easybuild/easyconfigs/p/python-xxhash/python-xxhash-3.4.1-GCCcore-12.3.0.eb

```
@@ -0,0 +1,30 @@
+easyblock = 'PythonBundle'
+
+name = 'python-xxhash'
+version = '3.4.1'
+
+homepage = 'https://github.com/ifduyue/python-xxhash'
+description = 'xxhash is a Python binding for the xxHash library by Yann Collet.'
+
+toolchain = {'name': 'GCCcore', 'version': '12.3.0'}
+
+builddependencies = [
+    ('binutils', '2.40'),
+]
+
+dependencies = [
+    ('Python', '3.11.3'),
+    ('xxHash', '0.8.2'),
+]
+
+use_pip = True
+sanity_pip_check = True
+
+exts_list = [
+    ('xxhash', version, {
+        'preinstallopts': 'XXHASH_LINK_SO=1',
```

better to retain the comment 

```suggestion
        'preinstallopts': 'XXHASH_LINK_SO=1',  # use xxHash library from EB
```